examples/sample-pub/publisher.js

```diff
@@ -104,6 +104,23 @@ function startFlowAuto() {
     });
     return;
   }
+
+  if (flow == 'smartbutton') {
+    whenReady(function(subsciptions) {
+      var subs = subsciptions;
+      whenDemoReady(function() {
+        var smartButton = document.querySelector('button.swg-smart-button');
```

Comment:
Added stricts CSS selector to ensure that publisher is using `button` component.